landing-page/content/common/project/multi-engine-support.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Multi-Engine Support"
+bookHidden: true
+url: multi-engine-support
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Multi-Engine Support
+
+Multi-engine support is a core tenant of Apache Iceberg.
+The community continuously improves Iceberg core library components to enable integrations with different compute engines that power analytics, business intelligence, machine learning, etc.
+Support of [Apache Spark](../../../docs/spark-configuration), [Apache Flink](../../../docs/flink) and [Apache Hive](../../../docs/hive) are provided inside the Iceberg main repository.
```

Comment:
Nit: "Support for" is more natural than "Support of"

Reply:
Or maybe be more specific: "Connectors for Spark, Flink, and Hive are maintained in the main Iceberg repository." Do we need to mention the Hive version so that it isn't confusing when support is built directly into Hive releases?

Reply:
> Do we need to mention the Hive version so that it isn't confusing when support is built directly into Hive releases?

I thought about that and was a bit hesitated, because people have been using even Hive 2.1 with Hive 3 jar and somehow things still work in various cases... I don't how backwards compatible Hive will be in the future.